data/remnant missions.txt

```diff
@@ -1553,12 +1552,17 @@ mission "Remnant: Face to Maw 1b"
 	landing
 	to offer
 		has "Remnant: Return the Samples 2: done"
+	to fail
+		has "Remnant: Face to Maw 1: done"
+		has "Remnant: Face to Maw 1: active"
+	to complete
+		has "Remnant: Face to Maw 1: declined"
 	source
 		system "Nenia"
 	destination "Aventine"
 	on complete
 		conversation
-			'When you walk into Plume's lab on Aventine you find him berried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
+			`When you walk into Plume's lab on Aventine you find him berried in some kind of holographic projection of your egg returning mission. You can see that he appears to be watching the video in slow-motion as he taps out notes on his datapad. After a few minutes he looks up and see you standing in the entryway. He quickly pulls himself out of the projection and hits what you assume must be the pause button.`
```

Comment:
"berried" should be "buried".

Reply:
Should already be fixed, but I'll check again. Thanks!